### ISSUE TITLE
Add CONTRIBUTING.md

### ISSUE THREAD

**Issue by arunvelsriram** [mailman-elixir/mailman#76]
Hi. Having a contributing guide will be really helpful for people who are interested in fixing bugs and adding features.

**Comment by skosch** [comment]
Agreed, always nice to have. Personally I think as long as tests pass and the code is reasonably sane, PRs are very welcome, bonus points for new tests and docs and `@spec`s. And if someone wanted to commit (ha!) more time I wouldn't hesitate giving them push access. @arunvelsriram Do you know of any project that has a good guide we can steal?

@kamilc, @Ch4s3, any thoughts?

**Comment by arunvelsriram** [comment]
@skosch Off the top of my head I would say [Terraform](https://github.com/hashicorp/terraform/blob/master/.github/CONTRIBUTING.md) and [Nevergreen](https://github.com/build-canaries/nevergreen/wiki/contributing) have a good contribution guide. Terraform's guide is very extensive so we can steal bits and pieces from it. 

I have seen other good guides as well but don't remember the project names.
